docker/include/networking_scripts/change_dhcp_range

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash -e
+#
+# Dynamically change DHCP lease range, requires killing and restarting
+# dnsmasq as per documentation (SIGHUP does not reload configuration file).
+LOCAL_IF=${LOCAL_IF:-$HOSTNAME-eth0}
+
+range_start=$1
+range_end=$2
+net_mask=$3
```

Comment:
is this really net_mask?  Isn't the value "16"?  That's not a netmask, that's a CIDR or some kind of prefix space.. "net_mask" typically refers to the full 255.255.0.0 form.

Reply:
What should it be named? prefix?